website/server/libs/slack.js

```diff
@@ -93,3 +93,51 @@ module.exports = {
   sendFlagNotification,
   sendSubscriptionNotification,
 };
+
+function sendSlurNotification ({
+  // authorEmail,
+  // author,
```

Comment:
This `author` line is failing because when you pass data to this function, you are using `user` as the object's name. If you change this line to `author: user,` it should work.